src/routes.py

```diff
@@ -103,14 +103,14 @@ def create_topic() -> str | Response:
         user_session = UserSession.get_user_session_by_session_id(session_id)
 
     current_user = None
```

Comment:
Seems like we don't need this line anymore.

And check the same thing for other routes as well.